### ISSUE TITLE
Create-react-app Tests not running

### ISSUE THREAD

**Issue by Nasafato** [orta/vscode-jest#85]
I'm using version 0.9.3 of react-scrips in CRA, which relies on Jest v18.1.

I run the tests using `npm test` (or `npm run test`), and the CLI works perfectly.

However, the extension doesn't seem to be recognizing that the tests are being run. I have empty green circles next to all of them, and, upon highlight, they all display: `Test has not run yet, due to Jest only running tests related to changes.`

Yet the tests are definitely running. I'm not sure what the problem is, since opening VSCode DevTools doesn't show any errors.

**Comment by sthuber90** [comment]
I have the same issue with version 0.9.4 of react-scripts. The jest plugin just keeps spinning and not returning any results. I also posted and issue at [create-react-app](https://github.com/facebookincubator/create-react-app/issues/1748), because the plugin is working, but only after running `npm run eject` on the project, which I would like to avoid to keep my project small and simple

**Comment by orta** [comment]
Sorry - been on my honeymoon, just started getting back to writing code.

I think this is caused by two things, which now has 2 PRs

* https://github.com/facebook/jest/pull/3212
* https://github.com/orta/vscode-jest/pull/90

Which fixes this 👍

**Comment by blipntime** [comment]
This is an automatically generated message.

jmucsi@yahoo-inc.com is no longer with Yahoo! Inc.

Your message will not be forwarded.

If you require assistance with a legal matter, please send a message to legal-notices@yahoo-inc.com

Thank you!

**Comment by mattiascaricato** [comment]
this should be working now? Do I have to change the config?

**Comment by seanpoulter** [comment]
Does that mean it's not working for you now @mattiascaricato?

**Comment by mattiascaricato** [comment]
yes @seanpoulter , is not working for me

**Comment by seanpoulter** [comment]
Since this issue is closed, could you please create a new issue with all the required info to create your problem? Please include OS info, the version of `create-react-app`, and any output from the Jest Output channel or the Dev Tools console (Help > Toggle Dev Tools).

FWIW I could not recreate your issue on a POSIX system:
```
mkdir 85
cd 85
yarn init -y
yarn add create-react-app
yarn create-react-app app
cd app
code .
```
This runs the one test OK.
